src/Chinchilla/Topologies/Model/TopologyBuilder.cs

```diff
@@ -103,19 +104,19 @@ public void Visit(IBinding binding)
                     {
                         // Reverse from/to for binding exchanges, because the parameters of ExchangeBind
                         // don't match QueueBind
-                        bindFunction = (from, to, keys) => m.ExchangeBind(to, from, keys);
+                        bindFunction = (from, to, keys, args) => m.ExchangeBind(to, from, keys, args);
```

Comment:
Looking at `QueueBind` and `ExchangeBind` in [IModelExtensions.cs](https://github.com/rabbitmq/rabbitmq-dotnet-client/blob/5d0377b4b0cf6e640ba2477f54ee76787813ccdf/projects/client/RabbitMQ.Client/src/client/api/IModelExtensions.cs), and also back when these were overloads rather than extension methods, it doesn't look like the parameter order has changed? Unless you meant for another reason?